app/services/ExpressionGenerator.coffee

```diff
@@ -0,0 +1,41 @@
+class ExpressionGenerator
+
+  randInclusive: (min, max) ->
+    Math.floor(Math.random() * (max - min + 1)) + min
+
+  genRandomDigit: (min, max) ->
+    @randInclusive(min, max).toString()
+
+  genRandomOperator: ->
+    switch @randInclusive 0, 2
+      when 0 then "+"
+      when 1 then "-"
+      when 2 then "*"
+      else "?"
+
+  generate: (length) ->
+    if length <= 0 
+      throw "Length cannot be less than 1"
```

Comment:
Since the error says "Length cannot be less than 1", I would have the logic mirror that error, so line 17 would be
```
if length < 1
```